File: basic math/java/isprime.java
import java.util.Scanner;
public class isprime {
    public static boolean prime (int n) {
        if(n==1) return false;
        for(int i=2;i*i<=n;i++) {
            if(n%i==0) return false;
        }
        return true;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        if(prime(n)) System.out.println("Prime Number");
        else System.out.println("Not Prime Number");
    }

}

File: basic math/javascript/is_prime.js
const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

function isPrime(number) {
    if (number < 2) {
        return false
    }
    for (let i = 2; i*i <= number; i++) {
        if (number % i === 0) {
            return false
        }
    }
    return true
}

rl.question('Enter a number: ', (number) => {
    number = parseInt(number)
    if(isPrime(number)){
        console.log(`${number} is prime`)
    } else {
        console.log(`${number} is not prime`)
    }
    rl.close()
})